Facilitate use of any regular ERB in database.yml (#46134)
rails/rails: activerecord/CHANGELOG.md

> *   Facilitate use of any regular ERB in database.yml when creating rake tasks.

Reviewer: ```suggestion
*   Allow any ERB in the database.yml when creating rake tasks.
```

>     Any ERB can be used in `database.yml` even if it accesses environment

Reviewer: ```suggestion
    Any ERB can be used in `database.yml` regardless of whether it accesses environment
```

    configurations.

>     Deprecates `config.active_record.suppress_multiple_database_warning`.

Reviewer: ```suggestion

    Deprecates `config.active_record.suppress_multiple_database_warning`.
```
